server/views/partials/detail/sentenceSummary.njk

```diff
@@ -26,24 +26,6 @@
           {% if summary.receptionDate %}
             {{ summaryListRow("Date of reception", summary.receptionDate | formatDate, "sentenceSummaryReceptionDate") }}
           {% endif %}
-          {% if sentenceSummary.effectiveSentence %}
```

Comment:
this is identical to the first section of the sentence information below.  Since we're now just all storing on one page then thought best to remove.